src/modules/makeTasks.js

```diff
@@ -0,0 +1,27 @@
+// eslint-disable-next-line no-unused-vars
+import createElement from './HtmlElement.js';
+
+let tasks = [
+  {
+    description: 'Laundry',
+    completed: true,
+    index: 1,
+  },
+  {
+    description: 'study',
+    completed: true,
+    index: 2,
+  },
+  {
+    description: 'play',
+    completed: true,
+    index: 3,
+  },
+];
+const taskcontainer = document.querySelector('to-do-list');
+tasks = tasks.sort((a, b) => (b.index - a.index));
+export default () => {
+  tasks.forEach((tasks) => {
+    createElement = ('li', 'task-item', 'task-item', `${tasks.description}`, taskcontainer);
+  });
+};
```

Comment:
- Hey, I can't see the populated task as your to-do doesn't show the tasks. please kindly fix this issue.

![Screenshot from 2022-09-29 01-18-16](https://user-images.githubusercontent.com/59397159/192911150-edd94948-1803-4fbd-9f6a-a24c9dd81954.png)